res/values/strings.xml

```diff
@@ -278,6 +278,7 @@ there are any).</string>
     <string name="np_desc_it_networks" translatable="false">ATM, GTT, AMAT, ACTV</string>
     <string name="np_desc_ontario">Ottawa, Toronto</string>
     <string name="np_desc_quebec">Deux-Montagnes, Laval, L\'Assomption, Outaouais, Sud-Ouest, Quebec, Haut-Saint-Laurent, Lanaudière, La Presqu\'Île, Laurentides, Montreal, Les Moulins, Vallée du Richelieu, Chambly-Richelieu-Carignan, Roussillon, Sorel-Varennes, Le Richelain, Sherbrooke, Sainte-Julie</string>
+    <string name="np_desc_oregon">Oregon, USA</string>
```

Comment:
I suggest you use that as a name (`setName()`) and use something for the description that somehow describes what is covered and what is not, available from here: https://navitia.opendatasoft.com/explore/dataset/us-or/table/?sort=type_file